src/main/scala/firrtl/LoweringCompilers.scala

```diff
@@ -150,7 +150,8 @@ class EmitVerilogFromLowFirrtl (val writer: Writer) extends Transform with Simpl
       passes.SplitExpressions,
       passes.CommonSubexpressionElimination,
       passes.DeadCodeElimination,
-      passes.VerilogRename)
+      passes.VerilogRename,
+      passes.VerilogPrep)
```

Comment:
It's not clear to me why `VerilogPrep` is present here. Is it different from lowering types?